done/static/css/done.css

```diff
@@ -94,4 +94,34 @@
 
 .done_nogrow {
     flex-grow: 0;
-}
\ No newline at end of file
+}
+
+/* Right to left (rtl) language css */
+
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-inner {
+    margin-right: 0;
+}
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-switch {
+    right: 0px;
+}
+
+body.rtl .done_onoffswitch-inner {
+    margin-right: -100%;
+}
+
+body.rtl .done_onoffswitch-inner:before {
+    content: "Unmark";
+    padding-right: 10px;
+    background-color: #EEEEEE;
```

Comment:
Why are there colors that are specified in RTL? I would expect the only difference in the RTL version to be related to left/right layout.